src/NServiceBus.Extensions.DependencyInjection/EndpointWithExternallyManagedServiceProvider.cs

```diff
@@ -0,0 +1,24 @@
+﻿namespace NServiceBus
+{
+    using Extensions.DependencyInjection;
+    using Microsoft.Extensions.DependencyInjection;
+
+    /// <summary>
+    /// Provides factory methods for creating endpoints instances with an externally managed container implementing Microsoft dependency injection abstraction.
+    /// </summary>
+    public static class EndpointWithExternallyManagedServiceProvider
```

Comment:
can't come up with a better name and it's technically correct so I'm 👍

Reply:
Agree 👍